### DISCUSSION TITLE
Welcome to Artistic Cinemagraph Discussions!

### DISCUSSION THREAD

**Discussion by text2cinemagraph** [text2cinemagraph/text2cinemagraph#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.

  To get started, comment below if you have questions or have tried and were able to generate some cool-looking cinemagraphs!

**Comment by shdrbs** [comment]
hello.
Is it possible to use Text2Cinemagraph to move only a specific part of a shirt image?
